updater/Dockerfile

```
@@ -0,0 +1,22 @@
+# Run the Hubble updater with Docker
+
+FROM python:alpine
+
+# Set environment variables
+ENV APP_HOME /hubble-updater
+
+# Create a directory for our application
+# and set it as the working directory
+RUN mkdir $APP_HOME
+WORKDIR $APP_HOME
+
+# Copy over our application code
+ADD reports $APP_HOME/reports
+ADD scripts $APP_HOME/scripts
+ADD ./check-schema.py ./helpers.py ./schema.py ./update-stats.py $APP_HOME/
```

Should we use a wildcard instead?
```suggestion
ADD ./*.py $APP_HOME/
```
I’m just wondering whether that would make it more future-proof (we may add or remove files later on, in which case this Dockerfile would break if we forgot to update it accordingly).